### PR TITLE
chore(docs): add selenium webdriver documents

### PR DESCRIPTION
**Pull request by cnishina**
Placeholders for api documentation.

### CODE REVIEW

File: lib/selenium-webdriver/locators.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
```
Comment (juliemr) [inline lib/selenium-webdriver/locators.js]: Should remove this license, since this will be our code.

File: lib/selenium-webdriver/locators.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+/**
+ * @fileoverview Factory methods for the supported locator strategies.
+ */
+
+goog.provide('webdriver.By');
```
Comment (juliemr) [inline lib/selenium-webdriver/locators.js]: We don't need these `goog.provide` things, do we?

File: lib/selenium-webdriver/webdriver.js [hunk 3]
```diff
@@ -0,0 +1,889 @@
...
+ * @param {!webdriver.WebDriver} driver The parent WebDriver instance for this
+ *     element.
+ * @param {!webdriver.promise.Promise.<!webdriver.WebElement>} el A promise
+ *     that will resolve to the promised element.
+ * @constructor
+ * @extends {webdriver.WebElement}
+ * @implements {webdriver.promise.Thenable.<!webdriver.WebElement>}
+ * @final
+ */
+webdriver.WebElementPromise = function(driver, el) {
```
Comment (juliemr) [inline lib/selenium-webdriver/webdriver.js]: I don't think we need this (or anything below) either.

File: website/docgen/dgeni-config.js [hunk 4]
```diff
@@ -71,14 +71,14 @@ myPackage.config(function(readFilesProcessor, templateFinder, writeFilesProcesso
...
-    {include: 'node_modules/selenium-webdriver/lib/webdriver/webdriver.js'},
-    {include: 'node_modules/selenium-webdriver/lib/webdriver/key.js'},
-    {include: 'node_modules/selenium-webdriver/lib/webdriver/promise.js'}
+    {include: 'built/element.js'},
+    {include: 'built/protractor.js'},
+    {include: 'built/locators.js'},
+    {include: 'built/expectedConditions.js'},
+    {include: 'built/selenium-webdriver/locators.js'},
+    {include: 'built/selenium-webdriver/webdriver.js'}
+    // {include: 'node_modules/selenium-webdriver/lib/webdriver/key.js'},
```
Comment (juliemr) [inline website/docgen/dgeni-config.js]: Don't we still want key and promise, for now? (Until we make a change to link to webdriver docs).

Comment (juliemr) [inline website/docgen/dgeni-config.js]: Nevermind - we're going to remove doc generation from the selenium-webdriver completely, and just add links later. Let's make an issue to capture this TODO.

File: lib/selenium-webdriver/locators.js (hunk 2 above)
Comment (cnishina) [inline lib/selenium-webdriver/locators.js]: Leaving it in for now. They helped link the website together.

File: lib/selenium-webdriver/locators.js (hunk 1 above)
Comment (cnishina) [inline lib/selenium-webdriver/locators.js]: License removed.

File: website/docgen/dgeni-config.js (hunk 4 above)
Comment (cnishina) [inline website/docgen/dgeni-config.js]: Added a TODO.

**Pull request merged**